foundation-extensions/mobile-core/README.md

```diff
@@ -1,34 +1,37 @@
 # Mobile Core
 
-The Mobile Core represents the core Adobe Experience Platform SDK that is required for every app implementation. The core contains a common set of functionality and frameworks, such as  Experience Cloud Identity services, data event hub, Rules Engine, reusable networking, disk access routines, and so on, which are required by all Adobe and third-party extensions. The following sections provide additional information about what is provided in the Mobile Core extension.
-
 {% hint style="warning" %}
 As of **April 1, 2020**, Apple will no longer support the UIWebView API. To avoid any issues, ensure that you are using iOS SDK version 2.3.4 or later. For more information about the UIWebView API, see [UIWebView ](https://developer.apple.com/documentation/uikit/uiwebview).
 {% endhint %}
 
+Mobile Core is mandatory for all Adobe Experience Platform app implementations. Mobile Core contains a common set of functionality and frameworks, including Experience Cloud Identity services, data event hub, Rules Engine, reusable networking, and disk access routines. The following sections provide additional information about what is provided in the Mobile Core extension.
+
 ## Mobile Core generic APIs
 
 The SDK provides core, extension-independent APIs to facilitate the event tracking of user screens, actions, and PII data. Events generated by these APIs are published to the SDK event hub and are available for use by extensions. For instance, when the Analytics extension is installed, all user actions and app screens event data is sent to the appropriate Analytics reporting endpoints.
 
-For more information, see [Mobile Core API Reference](mobile-core-api-reference.md) and [Mobile Core Event Reference](mobile-core-event-reference.md).
+For more information, please read the [Mobile Core API reference](mobile-core-api-reference.md).
 
 ## Configuration
 
-The configuration extension provides the events in the Event Hub that provide the configuration details for all installed extensions. For more information, see [Configuration](configuration/).
+The Configuration extension provides different APIs to set up the SDK's configuration settings. For more information, please read the documentation on the [Configuration extension](configuration/).
 
 ## Identity
 
-The Identity framework enables the Experience Cloud ID \(ECID\) service and is a required component for most SDK implementations. For more information, see [Identity](identity/).
+The Identity framework lets your app use Experience Cloud ID (ECID). Using ECIDs improves synchronization with Adobe and other customer identifiers. Please read the documentation on the [Identity extension](identity/) for more information.
```

Comment:
Should this have "for more information" at the front of the sentence like the others?